cli/tools/jupyter/server.rs

```diff
@@ -577,13 +595,13 @@ impl JupyterServer {
       connection
         .send(
           messaging::ExecuteReply {
-            execution_count: self.execution_count,
+            execution_count: self.execution_count.into(),
             status: ReplyStatus::Error,
-            error: Some(ReplyError {
+            error: Some(Box::new(ReplyError {
```

Comment:
Good catch on this. By the way I did this to pack the enum for jupyter message content more readily.